bin/zopen-init

```diff
@@ -597,7 +597,7 @@ init
 generateFileSystem
 generateZopenConfig
 ! ${refresh} && installPrereqs
-generateJsonConfiguration
+! ${refresh} && generateJsonConfiguration
```

Comment:
If I recall correctly, the reason for enabling this on refresh was to generate a json configuration on zopen update for users who did not have a config.json